app/src/main/java/edu/mit/mitmobile2/tour/utils/TourUtils.java

```diff
@@ -115,4 +117,16 @@ public int compare(Object lhs, Object rhs) {
         }
         return tourStops;
     }
+
+    public static String formatDistanceBySmoots(Double distance) {
```

Comment:
In the iOS app they don't show decimals for Smoots, so we can just show the integer representation